<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QGroupBox" name="ApType">
          <property name="title">
           <string>Ap Type</string>
          </property>
          <widget class="QRadioButton" name="radioButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>92</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>OMAP</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>92</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>EXYNOS 5250</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>92</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>EXYNOS 5420</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,50">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="info"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,50">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>x min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="xmin"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>x max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="xmax"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>y min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ymin"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>y max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ymax"/>
          </item>
          <item>
           <widget class="QLabel" name="MBperCountLabel">
            <property name="text">
             <string>KB / count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="MBperCount"/>
          </item>
          <item>
           <widget class="QPushButton" name="DefaultRange">
            <property name="text">
             <string>Default Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AverageMode">
            <property name="text">
             <string>Average Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="groupByLabel">
            <property name="text">
             <string>group by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="groupBy"/>
          </item>
          <item>
           <widget class="QPushButton" name="DefaultMode">
            <property name="text">
             <string>Default Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
